docs/use-cases/auctions/delegated-voucher-sdk/index.md

```diff
@@ -0,0 +1,37 @@
+# SDK for Delegated Voucher Auctions
+
+> Modular SDK for App developers supporting delegated voucher auctions
```

Comment:
An SDK is no use case?

I think this should be mentioned in the corresponding hydra-auction pages.